### DISCUSSION TITLE
Повышенная реакция антивирусов на версию 7.6.1 (Windows)

### DISCUSSION THREAD

**Discussion by birykvlad** [getlantern/lantern-forum-ru#10] [General]
Свежая версия 7.6.1 блокируется антивирусом Касперского (увы, выключить его не могу, рабочий ПК).
Результаты анализа VirusTotal версии 7.6.1 - https://www.virustotal.com/gui/file/5fe628fa58e5dc8044f6cb09cd2cba02b27226bc1d1431e0516d39a27a283c12?nocache=1

Результаты анализа VirusTotal версии 7.4.0 (более чистые) - https://www.virustotal.com/gui/file/e065e899c1e82dcb205e0e42be8ec94860fc7141339d67d8e714d0c247187ad2?nocache=1

**Comment by rtx2014** [comment]
7.6.2 такой же чистый, как и 7.4.0

**Reply by birykvlad** [reply]
Да, сейчас все в порядке. Свежий результат последней версии - https://www.virustotal.com/gui/file/3b91adffa027231c632cff0d715d3c768518e09f4b7b6e5199a36e8825adccdb?nocache=1
